explorer/requirements.txt

```diff
@@ -1,6 +1,26 @@
-osmnx==0.6
-shapely==1.6.1
-descartes==1.1.0
+descartes>=1.1.0
 pymongo==3.5.1
-geojson==2.3.0
+# Geo methods
```

Comment:
Maybe split into a second requirement file called explorer_requirements and import/install bother the backend reqs and the explorer reqs in the explorer docker? Otherwise it will get tricky to keep them in sync.